backend/src/common/types/task-note/task-note-get-all-arguments-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+import { EntityPaginationRequestQueryDto } from '~/common/types/types';
+
+type TaskNoteGetAllArgumentsDto = {
+  taskId: number;
+} & EntityPaginationRequestQueryDto;
```

Comment:
Why do we need bitwise operator here? I did not get the idea, could you explain, please?

Reply:
To get all notes, we need `taskId`, but I'm also paginating it, so I need `page` and `count`
I could've used:
```
type TaskNoteGetAllArgumentsDto = {
  taskId: number;
  page: number;
  count: number;
}
```
But we already have `EntityPaginationRequestQueryDto`, so no need to repeat the same code

Reply:
@WilhelmWesser https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types

Reply:
Thanks, I thought about that operator from the numerical side